Modules/ProfileManager/Profile.lua

```diff
@@ -19,6 +26,9 @@ function Profile:New(name, class, spec, main)
         patch = 0,
         changeset = ""
     }
+    o.alts = {}
+    o.altList = {}
+    o.altCount = 0
```

Comment:
I don't think this is clear (or needed).
As I understand it, both of these contain GUIDs. (This should be in the comment or variable name)
One is a dictionary allowing for efficient updating; the other is a list...

The alt count is definitely not needed, there are efficient ways to get the length of a list...

Not suggesting a fix yet because structure of the code is not clear to me yet.